scripts/requirements.txt

```diff
@@ -1,3 +1,3 @@
 requests==2.25.0
 readme-renderer[md]==29.0
-manim
+manim==0.15.0
```

Comment:
I feel better not to hardcode this. Instead, let it use the latest module available on PyPI. This way I could deploy the website after each release by just clicking on a button and not making a commit after each update. But if you could set up depandabot for this, in which case this would be fine.